Project_2/djangoProject/cars/migrations/0002_remove_car_car_maker_remove_car_car_model_and_more.py

```diff
@@ -163,6 +163,11 @@ class Migration(migrations.Migration):
             name='sale_price',
             field=models.FloatField(null=True, validators=[django.core.validators.MinValueValidator(0)]),
         ),
+        migrations.AddField(
+            model_name='car',
+            name='short_title',
+            field=models.CharField(default='', max_length=100),
+        ),
```

Comment:
No need to modify anything, but there is no need to regenerate the migrations every time a new field is added. Is is natural in a real-life product to not have the final version of the models from the start, as they might change as requirements do, and have multiple migration in different points in time that make modifications to the model. So, just add a new migration next time without deleting the previous ones and unapplying them.